### PR TITLE
Feature/17

### PR DESCRIPTION
**Pull request by Sylvain6**
TO TEST :
every single behaviour that seems you weird on the form, let's try any error on any field (actually we have no verification on email in front, side, i'll do it asap)
you have to make down then make install your backend to make it works
HAVE FUN GUYS 

HOW TO TEST :
every required field should display an error if empty
if you enter a bad adress, a specific error should be raised for it
you can't have two times the same email address
should be redirected to confirmation page after submit

### CODE REVIEW

File: api/config/packages/security.yaml [hunk 1]
```diff
@@ -23,13 +23,13 @@ security:
...
-            guard:
-                authenticators:
-                    - lexik_jwt_authentication.jwt_token_authenticator
+        # api:
+        #     pattern:  ^/api
+        #     stateless: true
+        #     provider: entity_provider
+        #     guard:
+        #         authenticators:
+        #             - lexik_jwt_authentication.jwt_token_authenticator
```
Comment (LouisHrg) [inline api/config/packages/security.yaml:32]: Why did you delete the auth system ?

File: api/src/DataFixtures/ShopFixtures.php [hunk 2]
```diff
@@ -43,10 +43,8 @@ public function load(ObjectManager $manager)
         $shop->setIsActive(true);
         $shop->setOpeningHours($businessHours);
 
-
-        $geocode = $this->faker->latitude(-90,90).', '.$this->faker->longitude(-180,180);
-
-        $shop->setGeocode($geocode);
+        $shop->setLatitude($this->faker->latitude(-90,90));
```
Comment (LouisHrg) [inline api/src/DataFixtures/ShopFixtures.php:46]: When I'll merge #38 you'll need to use the new latitude & longitude

File: api/src/DataFixtures/UserFixtures.php [hunk 3]
```diff
@@ -28,12 +28,13 @@ public function __construct(UserPasswordEncoderInterface $encoder)
     public function load(ObjectManager $manager)
     {
         // Create an admin user
-        $user = new User('admin');
+        $user = new User();
+        $user->setUsername("Nonnonononono");
+        $user->setFirstname("Richard");
```
Comment (LouisHrg) [inline api/src/DataFixtures/UserFixtures.php:33]: Rebase and use faker

File: api/src/Entity/User.php [hunk 4]
```diff
@@ -47,10 +47,24 @@ class User implements UserInterface
 
     /**
      * @Groups({"read", "write"})
-     * @ORM\Column(type="string", length=25, unique=true)
+     * @ORM\Column(type="string", length=255, unique=true)
```
Comment (LouisHrg) [inline api/src/Entity/User.php:50]: Limit to 100

File: front/src/layouts/LayoutDefault.vue [hunk 5]
```diff
@@ -9,6 +9,9 @@
     <el-menu-item route="/"  index="1">
       Home
     </el-menu-item>
+    <el-menu-item route="/register" index="2">
+      Register
```
Comment (LouisHrg) [inline front/src/layouts/LayoutDefault.vue:13]: later it'll be in another place

File: front/src/views/Register.vue [hunk 6]
```diff
@@ -0,0 +1,246 @@
+<template>
```
Comment (LouisHrg) [inline front/src/views/Register.vue:1]: flemme de review mdr

File: front/src/layouts/LayoutDefault.vue (hunk 5 above)
Comment (Sylvain6) [reply front/src/layouts/LayoutDefault.vue:13]: yes so lets place it here for now no ?

File: front/src/views/Register.vue (hunk 6 above)
Comment (Sylvain6) [reply front/src/views/Register.vue:1]: ahahahah tu fais bien

File: api/config/packages/security.yaml (hunk 1 above)
Comment (Sylvain6) [reply api/config/packages/security.yaml:32]: cause i wasn't able to work with it, as i don't know that much api platform i wasn't able to disable it on routes that i wanted too and i think it'll be the same for the others

**Comment by theosenoussaoui** [comment]
If I enter a bad adress, all the opening times and closing times are resetting.
